lib/utils.js

```diff
@@ -24,12 +24,42 @@ function sendSearchResponse(movies, resp) {
 
   resp
     .say(['Add', movies[0].original_title, '(' + movies[0].year + ')', 'to your list?'].join(' '))
-    .session('promptData', buildPrompt(movies.slice(0, 5)))
+    .session('promptData', buildPrompt(movies))
     .shouldEndSession(false)
     .send();
+
+function formatSearchResults(movies)
+{
```

Comment:
Sorry to be picky, but do you mind updating all these open curlies to be on the same line as the function? Just want the formatting to internally consistent in this file.